formulaire/src/main/resources/public/ts/controllers/form-editor.ts

```diff
@@ -773,7 +773,9 @@ export const formEditorController = ng.controller('FormEditorController', ['$sco
                         formElement.choices.replaceSpace();
                         let positionCounter: number = 1;
                         for (let choice of formElement.choices.all) {
-                            if (choice.value && !registeredChoiceValues.find((v: string) => v === choice.value)) {
+                            if (!choice.value && choice.id) {
+                                await questionChoiceService.delete(choice.id);
+                            } else if (choice.value && !registeredChoiceValues.find((v: string) => v === choice.value)) {
```

Comment:
A voir si c'est possible
```suggestion
                            } else if (choice.value && !registeredChoiceValues.some((v: string) => v === choice.value)) {
```